content/en/docs/Installation/_index.md

```diff
@@ -40,23 +40,31 @@ helm repo update
 
 Use Helm 3.2+ to create a Namespace and install Kyverno.
 
+Beginning with Helm 1.5.0, the Kyverno CRDs must be added seperately and before Kyverno is installed.
```

Comment:
Should we change it to 1.4.2 and merge the PR since the Chart has been changed already? The current installation guidance was broken due to missing CRDs.

Reply:
This speaks of "Helm 1.5.0" and not "Kyverno 1.5.0".

Reply:
I adjusted version to 1.4.2 , fixed reference to be Kyverno and also mention which version of Helm chart begins this change since Helm chart and Kyverno are not same version number.